src/Sarif.FunctionalTests/ConverterTestData/Semmle/results.csv.sarif

```diff
@@ -0,0 +1,1353 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-1.0.0",
+  "version": "1.0.0",
+  "runs": [
+    {
+      "tool": {
+        "name": "Semmle"
+      },
+      "results": [
+        {
+          "level": "warning",
```

Comment:
Fixed to not emit "warning". I don't think we have default handling _per se_ in the SDK. I think each producer is deciding whether to emit default values.

---
In reply to: [90075506](https://github.com/Microsoft/sarif-sdk/pull/545#discussion_r90075506) [](ancestors = 90075506)